static/styles/portico-signin.scss

```diff
@@ -383,13 +400,18 @@ html {
     border: 1px solid hsl(0, 0%, 53%);
 }
 
+.new-style .input-box textarea:focus {
+    border: 1px solid hsl(0, 0%, 53%) !important;
```

Comment:
Why do we need the `!important` here and not with the similar rules above?

Reply:
That's because it's overruled by the bootstrap.css. The other option is -
```css
.new-style .input-box input[type=text]:focus,
.new-style .input-box input[type=email]:focus,
.new-style .input-box input[type=password]:focus,
.new-style .input-box textarea:focus:required:invalid:focus {
    border: 1px solid hsl(0, 0%, 53%);
}
```